api/src/modules/proxy/proxy.controller.ts

```diff
@@ -16,10 +16,9 @@ import { Request, Response } from 'express';
 
 @UseGuards(JwtAuthGuard)
 @ApiBearerAuth()
-@ApiTags('AdminArea vector tile proxy')
+@ApiTags('Vector tile proxy')
```

Comment:
good point 😄 

if we add more/unrelated (to tiles) proxy endpoints in the future we may want to *then* split things across this and another controller, all using the same service and grouped under the same module, but just a thought for the future - maybe we won't need any of this 😅

Reply:
haha yes, I though we could add all the tiles endpoints under the same proxy and potentially split things if we add something not tile related.